src/hal/i_components/knob2float.icomp

```diff
@@ -34,8 +34,8 @@ license "GPL";
 
 FUNCTION(_)
 {
-    long delta_counts;
-    double tmp_out;
+    hal_float_t delta_counts;
```

Comment:
Presume you mean line 43, all the vars on 45 are hal_float_t
The component is not satisfactory in that respect, has always mixed s32 with float